app/utils/review.js

```diff
@@ -92,7 +93,9 @@ class ReviewApp {
 		}
 		this.positiveEventCount += 1;
 		if (this.positiveEventCount === numberOfPositiveEvent) {
-			tryReview();
+			if (!isFDroidBuild) {
```

Comment:
IMO you should test for the build and return right from the beginning of this function.